src/index.html

```diff
@@ -11,6 +11,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <main class="main">
+      <form
+        action="#"
+        method="post"
+        class="search-form--big"
+        data-qa="big"
+      >
+        <label class="search-form__label--big">
+          <img
+            src="images/Search.svg"
+            class="Icon__big"
+            alt="Icon"
+          >
+          <input
+            type="text"
+            class="search-form__input--big"
+            placeholder="Try &quot;Los Angeles&quot;"
+            data-qa="keypress"
+          >
+        </label>
+      </form>
```

Comment:
you can add block selector to each of your tags to move common rules to these selectors. 
So in this way you don't need to use styles like this

```
.search-form__input--big,
.search-form__input--small {
  border-radius: var(--border-radius);
  border: var(--border);
  background: var(--background);
  box-sizing: var(--box-sizing);
  color: var(--color);
  font-family: inherit;
}
```

You will just have a common block selector
```
search-form__input
```


```suggestion
      <form
        action="#"
        method="post"
        class="search-form search-form--big"
        data-qa="big"
      >
        <label class="search-form__label search-form__label--big">
          <img
            src="images/Search.svg"
            class="icon icon__big"
            alt="Icon"
          >
          <input
            type="text"
            class="search-form__input search-form__input--big"
            placeholder="Try &quot;Los Angeles&quot;"
            data-qa="keypress"
          >
        </label>
      </form>
```